src/components/Explore/TokenTable/SearchBar.tsx

```diff
@@ -0,0 +1,77 @@
+import { useAtom } from 'jotai'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+import { MEDIUM_MEDIA_BREAKPOINT } from '../constants'
+import { filterStringAtom } from '../state'
+import searchIcon from './search.svg'
```

Comment:
These assets should be under assets, not in the component folder.

Reply:
@kayleegeorge can you move this file